src/dayview/dayview.ts

```diff
@@ -141,18 +141,43 @@ export async function dayview(args: DayviewCmdArgs): Promise<string> {
     .attr("height", dataHeight)
     .attr("x", labelWidth);
 
-  await Promise.all(
+  // Add debugging for data loading
+  const dayResults = await Promise.all(
     days.map(async (date, i) => {
       const y = i * (dayHeight + interdayMargin);
-      const daySvg = await singleDay({
-        db,
-        date,
-        dataWidth,
-        height: dayHeight,
-        labelWidth: 0,
-      });
-
-      dataArea.append(() => daySvg).attr("y", y);
+      try {
+        const daySvg = await singleDay({
+          db,
+          date,
+          dataWidth,
+          height: dayHeight,
+          labelWidth: 0,
+        });
+        return { date, svg: daySvg, y };
+      } catch (error) {
+        console.error(`Error loading data for ${date}:`, error);
+        return { date, error, y };
+      }
+    }),
+  );
+
+  // Debug errors
+  const allErrors = dataArea.selectAll(".error");
+
+  if (allErrors.size() > 0) {
+    console.log("Error details:");
+    const erroredFields = new Set<string>();
+    allErrors.each(function () {
+      const field = d3.select(this).attr("field");
+      console.log("Error field:", field);
```

Comment:
For now, there is already a `console.warn` emitted by the checkCmd, which is what is checking for the errors.
see fieldSvgBlocks.ts:131